Makefile

```diff
@@ -354,6 +354,11 @@ ifneq (${FIP_ALIGN},0)
 FIP_ARGS += --align ${FIP_ALIGN}
 endif
 
+# If ERROR_DEPRECATED is defined, use cert_create built with generic TBBR
+ifeq (${ERROR_DEPRECATED},1)
+USE_TBBR_DEFS := 1
+endif
+
```

Comment:
The problem with this is that it overrides any setting of USE_TBBR_DEFS in platform.mk.

Perhaps you could remove the initialization of USE_TBBR_DEFS in defaults.mk (as this is a special case) and then initialize here only in the case that it has not been previously defined.

So:

```
USE_TBBR_DEFS              ?= $(ERROR_DEPRECATED)
```